static/components/TaskDetails.jsx

```diff
@@ -8,21 +8,21 @@ import PollingMixin from "../mixins/polling";
 import TaskSummary from "./TaskSummary";
 import TimeSince from "./TimeSince";
 import { browserHistory } from 'react-router';
+import PropTypes from 'prop-types';
 
 var moment = require('moment');
 
 var Progress = React.createClass({
-  propTypes: {
-    value: React.PropTypes.number.isRequired,
-  },
-
   render() {
     return (
       <span className="progress" style={{width: this.props.value + '%'}} />
     );
   }
 });
 
+Progress.propTypes = {
```

Comment:
```Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.```
I did that to stop this annoying propTypes warning.  Might not be 100% necessary (yet?), but made my life easier when debugging stuff.